packages/world/ts/getWorldAbi.ts

```diff
@@ -26,8 +26,16 @@ export async function getWorldAbi({
   });
   const baseFunctionSelectors = (IBaseWorldAbi as Abi).filter(isAbiFunction).map(toFunctionSelector);
   const worldFunctionsAbi = worldFunctions
-    .map((func) => functionSignatureToAbiItem(func.signature))
-    .filter((abiItem) => !baseFunctionSelectors.includes(toFunctionSelector(abiItem)));
+    .map((func) => {
+      try {
+        return functionSignatureToAbiItem(func.signature);
+      } catch (error) {
+        console.error(error);
+      }
+    })
+    .filter(
+      (abiItem): abiItem is AbiFunction => !!abiItem && !baseFunctionSelectors.includes(toFunctionSelector(abiItem)),
```

Comment:
tiny nit but for consistency across the codebase, could we use `isDefined` here? (from `@latticexyz/common/utils`)